### ISSUE TITLE
 Dark Fuchsia does not show red mark on the scroll bar

### ISSUE THREAD

**Issue by schakalakka** [thvardhan/Gradianto#9]
The scroll bar on the right shows errors, warnings and more. 
Clear coding errors are not marked red but yellow instead. 
This is a bit misleading. 

Should it be fixed? Or am I the only one with this issue?

**Comment by schakalakka** [comment]
I think the ERROR_STRIPE_COLOR in the ERRORS_ATTRIBUTES is not set properly. 
Instead of bcb933 (yellowish) it should be something like bc2f2c (red).

```
<option name="ERRORS_ATTRIBUTES">
      <value>
        <option name="EFFECT_COLOR" value="bc2f2c" />
        <option name="ERROR_STRIPE_COLOR" value="bcb933" />
        <option name="EFFECT_TYPE" value="2" />
      </value>
</option>
```

**Comment by thvardhan** [comment]
Fixed in 3.7. The update should go live in 3-4 days and should show in the IntelliJ plugin window. Thanks for the report!
